pkg/xds/bootstrap/export_template.go

```diff
@@ -0,0 +1,5 @@
+package bootstrap
+
+var (
+	DnsLookupFamilyFromXdsHost = dnsLookupFamilyFromXdsHost
+)
```

Comment:
This is not a pattern we've been following much atm. So I think it makes sense for @jakubdyszkiewicz to choose whether we let this through or not WRT code coherency :).

Reply:
pinging @jakubdyszkiewicz, this is the approach that go std lib uses https://go.dev/src/net/http/export_test.go so I thought it's ok, happy to change it.

Reply:
You're right, I went with `2`